clash-prelude/tests/Clash/Tests/Resize.hs

```diff
@@ -1,18 +1,37 @@
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE TypeFamilies #-}
 
-module Clash.Tests.Resize (tests) where
+{-# OPTIONS_GHC -fplugin=GHC.TypeLits.Extra.Solver #-}
+{-# OPTIONS_GHC -fplugin=GHC.TypeLits.Normalise #-}
+{-# OPTIONS_GHC -fplugin=GHC.TypeLits.KnownNat.Solver #-}
 
+module Clash.Tests.Resize (tests, main) where
+
+import Clash.XException (XException)
 import Control.DeepSeq (NFData)
 import Control.Exception (SomeException, try, evaluate)
-import Clash.XException (XException)
 import Data.Either (isLeft)
 import Data.Proxy (Proxy(Proxy))
-import GHC.TypeNats (KnownNat, type (<=))
-import Test.Tasty (TestTree, testGroup)
-import Test.Tasty.QuickCheck
+import GHC.TypeNats (KnownNat, SomeNat(..), type (<=), type (+), Nat, Natural, someNatVal)
+import Hedgehog ((===))
+import Test.Tasty (TestTree, testGroup, defaultMain, adjustOption)
+import Test.Tasty.Hedgehog (HedgehogTestLimit(..), testPropertyNamed)
+import Test.Tasty.QuickCheck (testProperty, discard, ioProperty, counterexample)
+
+import qualified Test.Tasty.QuickCheck as Q
+import qualified Hedgehog as H
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
 
 import qualified Clash.Class.Resize as Resize
-import Clash.Sized.Index
+import Clash.Promoted.Nat (SNat(..))
+import Clash.Prelude (Unsigned, Signed, BitVector, Index)
```

Comment:
Did you intend to sort the imports like this? I got the sense you were separating non-qualified and qualified imports but here it's non-qualified again?

Reply:
No rhyme or reason really :see_no_evil:. I'll fix it.

Reply:
I want [`by-scope-then-qualified`](https://github.com/fourmolu/fourmolu/pull/403) and never manually sort imports again :heart_eyes: